gpbft/gpbft.go

```diff
@@ -69,6 +69,8 @@ type GMessage struct {
 	Ticket Ticket
 	// Justification for this message (some messages must be justified by a strong quorum of messages from some previous step).
 	Justification *Justification
+	// SupplementalData is the additional data to aid validation.
+	SupplementalData SupplementalData
```

Comment:
`SupplementalData` is already part of the Vote

Reply:
doh; thanks.